lib/web_ui/lib/src/engine/compositor/recording_canvas.dart

```diff
@@ -15,7 +15,12 @@ class SkRecordingCanvas implements RecordingCanvas {
   bool _hasArbitraryPaint = true;
 
   @override
-  int saveCount = 0;
+  int get saveCount => skCanvas.callMethod('getSaveCount');
+
+  set saveCount(int newSaveCount) {
+    throw UnimplementedError(
+        "The Skia backend doesn't support setting saveCount");
+  }
```

Comment:
Feel free to remove it entirely.

Reply:
I got rid of the `saveCount` setter, but I still needed a settable `_saveCount` in `RecordingCanvas`.